scripts/configure-hypervisor.sh

```diff
@@ -398,13 +398,26 @@ generate_qemu_options() {
 	# Support Ceph RADOS Block Device (RBD)
 	[ -z "${static}" ] && qemu_options+=(functionality:--enable-rbd)
 
+
+	if [ "${qemu_version_major}" -ge 5 ]; then
+		if [ "${qemu_version_minor}" -ge 2 ]; then
```

Comment:
This looks wrong since:

| qemu version | condition true ? |
|-|-|
| 5.1 | no |
| 5.2 ... 5.9 | yes |
| 6.0 | no |
| 6.1 | no |
| 6.2 ... 6.9 | yes | 

Can you use the safer `bc(1)` logic as used for `vxhs` on line 376?

Reply:
Hey I will move this to a separated PR :)